### PR TITLE
Add comment about default http_address

### PR DESCRIPTION
**Pull request by pablopla**
## Checklist:

- [ ] My change requires a change to the documentation or CHANGELOG.
- [ ] I have updated the documentation/CHANGELOG accordingly.
- [ ] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: contrib/oauth2_proxy.cfg.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## OAuth2 Proxy Config File
 ## https://github.com/pusher/oauth2_proxy
 
-## <addr>:<port> to listen on for HTTP/HTTPS clients
+## <addr>:<port> to listen on for HTTP/HTTPS clients. defaults to "127.0.0.1:4180"
```
Comment (JoelSpeed) [inline contrib/oauth2_proxy.cfg.example]: Nit: I think after the full stop we should use a capital
```suggestion
## <addr>:<port> to listen on for HTTP/HTTPS clients. Defaults to "127.0.0.1:4180"
```

File: contrib/oauth2_proxy.cfg.example [hunk 2]
```diff
@@ -10,7 +10,7 @@
 # tls_key_file = ""
 
 ## the OAuth Redirect URL.
-# defaults to the "https://" + requested host header + "/oauth2/callback"
+# defaults to "https://" + requested host header + "/oauth2/callback"
```
Comment (JoelSpeed) [inline contrib/oauth2_proxy.cfg.example]: Nit: Other comments in this example use `##` to start the line, separating the comments from the example config lines that are commented out, we should stay consistent
```suggestion
## defaults to "https://" + requested host header + "/oauth2/callback"
```

**Comment by JoelSpeed** [comment]
@pablopla I can't merge my suggestions into your branch, could you merge the suggestions and I'll merge this change in, thanks

**Comment by steakunderscore** [comment]
@pablopla Are you able to add Joel's suggestions. Then we can get your changes merged.

**Pull request closed without merging**